packages/themes/scss/_tokens.scss

```diff
@@ -5,322 +5,269 @@
 // This source code is licensed under the Apache-2.0 license found in the
 // LICENSE file in the root directory of this source tree.
 //
-
 @import './theme-maps';
-
 /// Primary interactive color; Primary buttons
 /// @type Color
 /// @access public
 /// @group @carbon/themes
-$interactive-01: map-get($carbon--theme, interactive-01) !default;
-
+$interactive-01: map-get($carbon--theme, 'interactive-01');
```

Comment:
Removal of `!default` here seems not intentional, is it correct...?

Reply:
Thanks @asudoh! Will make sure to add in 😄